build.sbt

```diff
@@ -84,6 +89,11 @@ val frontend = project
     fork in run := true,
     fork in Test := true,
     parallelExecution in test := false,
+    libraryDependencies ++= List(
+      "io.monix" %% "monix" % "2.3.2",
+      // Depend on ipcsocket only for testing since it has special changes to use clients
+      "org.scala-sbt.ipcsocket" % "ipcsocket" % "1.0.0" % Test
```

Comment:
ipcsocket modifies the local sockets implementations from Nailgun and adds a few utils to spawn clients, we're adding it here only in the tests so that we can create clients for named pipes and unix sockets.